### ISSUE TITLE
Отличия pip и GitHub версий

### ISSUE THREAD

**Issue by apugachev** [alexeykosh/lingcorpora.py#18]
Файлы библиотеки, устанавливаемые через pip install отличаются от тех, которые находятся в репозитории на GitHub

**Comment by agricolamz** [comment]
Да, последняя версия переехала [сюда](https://github.com/lingcorpora/lingcorpora.py)
